packages/application-shell/src/components/authenticated/oidc-callback-error-page.tsx

```diff
@@ -51,8 +51,8 @@ const AuthCallbackErrorPage = (props: TProps) => {
       {({ locale, messages }) => (
         <ConfigureIntlProvider locale={locale} messages={messages}>
           <ThemeProvider
-            theme="default"
-            themeOverrides={themesOverrides.default}
+            theme="recolouring"
```

Comment:
Should we change it first in UI Kit? Then we can keep this as `default`.

Reply:
Are we good with proceeding this way and doing the clean up as a follow up?

Reply:
Clarified in Slack. All good